### PR TITLE
Backward Compatibility

### PR DESCRIPTION
**Pull request by AbuBaniaz**
Add backward compatibility for show bouquets on launch.

### CODE REVIEW

**Comment by ccs-ccs** [comment]
I don't think the second "config." should be in line 1322...

upgrade(config.epgselection.grid.showbouquet, "config.epgselection.graph_showbouquet")

**Comment by AbuBaniaz** [comment]
Oops. At least we are on right track.

**Comment by ccs-ccs** [comment]
Thanks, it works ok.

File: lib/python/Components/UsageConfig.py [hunk 1]
```diff
@@ -1274,7 +1274,7 @@ def upgrade(configItem, name, valuemap = None, mapper = None):
 		print "[UsageConfig] Upgrading EPG settings"
 
 		okMap = {"Zap + Exit": "zapExit", "Zap": "zap"}
-		infoMap = {"Channel Info": "openEventView", "Single EPG": "openSingleEPG"}
+		infoMap = {"Event Info": "openEventView", "Single EPG": "openSingleEPG"}
```
Comment (SimonCapewell) [inline lib/python/Components/UsageConfig.py]: Don't think this change is actually correct. This should be the old value, which used to _Channel Info_

Comment (AbuBaniaz) [reply lib/python/Components/UsageConfig.py]: OK, I'll change it. Or you can just make the other change your end and close this request

Comment (AbuBaniaz) [reply lib/python/Components/UsageConfig.py]: Changed my end.

**Comment by SimonCapewell** [comment]
I've incorporated this with another rebase. It was an incorrect rebase yesterday that removed the 2 showbouquet upgrades.

**Comment by ccs-ccs** [comment]
I've just tried Abu's latest commit and it still looks ok from my end.

I'll leave it now until I see the files published on the forum.

Cheers

**Pull request closed without merging**